remote/config.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="icon" href="../../favicon.ico">
+
+    <title>Smart Mirror Remote</title>
+
+    <style>
+      @import url("https://gitcdn.xyz/repo/joshfire/jsonform/master/deps/opt/bootstrap.css");
+    </style>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+    crossorigin="anonymous">
+<link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+        <div class="container">
+            <div class="navbar-header">
+                <a class="navbar-brand" href="/">Smart Mirror Remote</a>
+            </div>
+            <a class="navbar-text navbar-config" href="/config.html"><span class="glyphicon glyphicon-cog" aria-hidden="true" aria-label="settings"></span></a>
+        </div>
+    </nav>
+
+    <!-- Begin page content -->
+    <div class="container">
+        <div class="row">
+            <div id="result" class="col-xs-12">
+                <h1 class="text-center">Loading...</h1>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-xs-12">
+                <p class="alert" id="out"></p>
+            </div>
+        </div>
+    </div>
+
+    <footer class="footer">
+        <div id="connection-bar" class="container">
+            <p id="connection-text" class="text-muted">Connecting...</p>
+        </div>
+    </footer>
+
+
+    <!-- Smart Mirror remote JavaScript
+    ================================================== -->
+    <!-- Placed at the end of the document for profit -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery.min.js"><\/script>')</script>
```

Comment:
`"../../assets/js/vendor/jquery.min.js"` doesn't exist anymore, so I'm going to get rid of it.